raiden/api/v1/encoding.py

```diff
@@ -35,16 +35,15 @@
 )
 
 
-# type converter for the flask route
 class HexAddressConverter(BaseConverter):
-
     def to_python(self, value):
-        value = address_decoder(value)
-        return value
+        if value[:2] == '0x':
```

Comment:
I have no strong argument in favor of it other than it's my preference to be stricter, relaxing rules is backwards compatible the opposite is not (e.g. geth's change in the address encoding for the genesis block)